src/actions/checkForUpdate.js

```diff
@@ -4,34 +4,34 @@ import DependencyGraph from '../lib/DependencyGraph';
 
 import type {PackageJSONType} from '../utils';
 
-export default async function checkForUpdate(
-  packageDefinition: PackageJSONType,
-  prompt: any
-) {
-  const result = await prompt(['library name', 'version']);
-  const packageToUpdate = result['library name'];
-  const version = result.version;
-  const depsList = Object.keys(packageDefinition.dependencies)
-    .map(dependency => [dependency, packageDefinition.dependencies[dependency]])
-    .filter(([dependency]) => dependency !== packageToUpdate);
-  const depsMap = new Map(depsList);
-  const dependencyGraph = new DependencyGraph(depsMap);
-  console.log('Building dependency graph...');
-  await dependencyGraph.resolveDependencyGraph();
-  dependencyGraph.removePackagesWithout(packageToUpdate);
-  const availableUpdates = dependencyGraph.packagesToUpdate(
-    packageToUpdate,
-    version
-  );
-  if (availableUpdates.length) {
-    console.log('You can update to', availableUpdates.join(', '));
-  } else {
-    console.log(
-      'Im afraid that you can not update',
-      packageToUpdate,
-      'to version',
-      version
+export default async function checkForUpdate(packageDefinition: PackageJSONType,
+                                             prompt: any,
```

Comment:
use eslint please

Reply:
You might want to add your .eslintrc so all can have the same configuration.

Reply:
should we use husky + prettier instead? That's what I do on my machine. no configurations, and auto-rewrite on every commit 👌

I'll create a PR for this